src/it/java/teammates/it/storage/sqlapi/AccountRequestsDbIT.java

```diff
@@ -88,4 +88,123 @@ public void testUpdateAccountRequest() throws Exception {
                 accountRequest.getEmail(), accountRequest.getInstitute());
         verifyEquals(accountRequest, actual);
     }
+
+    @Test
+    public void testSqlInjectionInCreateAccountRequestEmailField() throws Exception {
+        ______TS("SQL Injection test in email field");
+
+        // Attempt to use SQL commands in email field
+        String email = "email'/**/OR/**/1=1/**/@gmail.com";
```

Comment:
If this is to test SQL injection in `INSERT`, I think we would need to add closing parentheses somewhere? This applies for the other `testSqlInjectionInCreateAccountRequest*Field` methods.

Reply:
Actually, I was thinking, maybe it's fine...? If it's possible to do SQL injection here, the SQL query would be invalid, and an error would be raised. Since no error was raised, maybe it's sufficient to show that SQL injection is not possible? So maybe what I said about the closing parenthesis is not necessary.

Reply:
> Actually, I was thinking, maybe it's fine...? If it's possible to do SQL injection here, the SQL query would be invalid, and an error would be raised. Since no error was raised, maybe it's sufficient to show that SQL injection is not possible? So maybe what I said about the closing parenthesis is not necessary.

Agreed and resolving as per our chat discussion